molecule/cookiecutter/molecule/{{cookiecutter.role_name}}/{{cookiecutter.molecule_directory}}/{{cookiecutter.scenario_name}}/molecule.yml

```diff
@@ -19,6 +19,7 @@ provisioner:
   name: {{ cookiecutter.provisioner_name }}
   lint:
     name: {{ cookiecutter.provisioner_lint_name }}
+    enabled: false
```

Comment:
? Also for https://github.com/ansible/ansible/issues/63734?

Reply:
Shroud we include a comment with reasoning? Maybe the team will reconsider that decision  under peer-pressure? Anyway we are going to rewrite the linter logic anyway and probably deprecate these options in near future.

Reply:
Comment would be good.